templates/redis.conf.j2

```diff
@@ -0,0 +1,1112 @@
+# Redis configuration file example.
+#
```

Comment:
At least add `ansible_managed` to this monster.

Reply:
done